server/plugins/api.js

```diff
@@ -1,13 +1,15 @@
 const fetch = require('node-fetch')
+const config = require('../config')
 
 module.exports = {
   plugin: {
     name: 'api',
     register: (server, options) => {
       server.method({
         name: 'api.getJson',
-        method: async (url) => new Promise(async (resolve, reject) => {
+        method: async (action) => new Promise(async (resolve, reject) => {
           try {
+            const url = config.apibase + action
             const response = await fetch(url)
             resolve(await response.json())
           } catch (error) {
```

Comment:
As the `method` signature is async the function implicitly returns a promise.
You could rewrite the `method` function as:
```
        method: async action => {
          const response = await fetch(config.apibase + action)
          return response.json()
        },
```